lib/primitives/address.js

```diff
@@ -4,15 +4,15 @@
  * https://github.com/handshake-org/hsd
  */
 
-'use strict';
+"use strict";
 
-const assert = require('bsert');
-const bio = require('bufio');
-const {bech32} = require('bstring');
-const blake2b = require('bcrypto/lib/blake2b');
-const sha3 = require('bcrypto/lib/sha3');
-const Network = require('../protocol/network');
-const consensus = require('../protocol/consensus');
+const assert = require("bsert");
+const bio = require("bufio");
+const { bech32 } = require("bstring");
+const blake2b = require("bcrypto/lib/blake2b");
+const sha3 = require("bcrypto/lib/sha3");
+const Network = require("../protocol/network");
+const consensus = require("../protocol/consensus");
```

Comment:
hello wi-ski, thank you for submitting a pull request. If you notice in all of the other files, they use `'` over `"` so it may be a good idea to revert them back to `'`

Reply:
Oops, totally. Will fix